assemble/bin/accumulo-cluster

```diff
@@ -66,7 +66,7 @@ function parse_config {
 
   trap 'rm -f "$CONFIG_FILE"' EXIT
   CONFIG_FILE=$(mktemp) || exit 1
-  ${accumulo_cmd} org.apache.accumulo.core.conf.cluster.ClusterConfigParser "${conf}"/cluster.yaml >"$CONFIG_FILE" || parse_fail
+  ${accumulo_cmd} org.apache.accumulo.core.conf.cluster.ClusterConfigParser "${conf}"/cluster.yaml "$CONFIG_FILE" || parse_fail
```

Comment:
Instead of just relying on the number of args, it would be better to have an explicit flag like `-o outputfile`. However, this is an internal tool, not public API, so it really doesn't matter as long as it works for us. I mention this here, because I think it's a good principle to keep in mind for any similar changes in future that might apply to a user facing command.